File: src/main/java/no/nav/domain/pensjon/regler/beregning/penobjekter/TilleggTilHjelpIHuset.java
package no.nav.domain.pensjon.regler.beregning.penobjekter;

import no.nav.domain.pensjon.regler.beregning.Ytelseskomponent;
import no.nav.domain.pensjon.regler.kode.YtelsekomponentTypeCti;

import java.io.Serializable;

/**
 * Brukes kun av BEF270 til G-omregning.
 * 
 * @author Andreas Nilsen
 */
public class TilleggTilHjelpIHuset extends Ytelseskomponent implements Serializable {

    

    private int grunnlagForUtbetaling;


    public TilleggTilHjelpIHuset(int grunnlagForUtbetaling) {
        super();
        this.grunnlagForUtbetaling = grunnlagForUtbetaling;
        ytelsekomponentType = new YtelsekomponentTypeCti("HJELP_I_HUS");
    }


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning/penobjekter/TilleggTilHjelpIHuset.kt
package no.nav.pensjon.regler.domain.beregning.penobjekter

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent
import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum
import java.io.Serializable

/**
 * Brukes kun av BEF270 til G-omregning.
 */
class TilleggTilHjelpIHuset : Ytelseskomponent(), Serializable {
    var grunnlagForUtbetaling = 0

    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.HJELP_I_HUS

}